server.R

```diff
@@ -1132,3 +1132,4 @@ server <- function(input, output, session) {
   })
   
 }
+
```

Comment:
Thank you for this PR @darshanramani 

Looks like your changes were overwritten by the latest commit. I can only see an empty line as part of the changes now. Can you add back the changes you have made in previous commits and update this PR again?

Also, please `sync` your fork before making changes so that you work with the latest code. We have updated the instructions on the README for the same. Ex: https://github.com/Mindshift-Analytics/roicalculator/blob/854365733b36e3f923d9dc6c30641f276ecb8caf/README.md#2-sync-your-branch-with-main-regularly